include/seastar/core/metrics.hh

```diff
@@ -680,6 +680,14 @@ impl::metric_definition_impl make_total_operations(metric_name_type name,
     return make_counter(name, std::forward<T>(val), d, labels).set_type("total_operations");
 }
 
+/*!
+ * \brief Update the aggregation labels of a metric family
+ */
+void update_aggregate_labels(const group_name_type& group_name,
+                             const metric_name_type& metric_name,
+                             const std::vector<label>& aggregate_labels,
+                             int handle = default_handle());
```

Comment:
Having read your Redpanda PR, we only aggregate on the default handle (`/metrics`). If we remove the final defaulted arg. Could we upstream this change?

Reply:
Yes I think so. I will definitely try once everything is agreed on and merged.

All the "default_handle" stuff is just from us anyway right?

Reply:
@VladLazar Actually I realize now I might have misunderstood your first comment.

Did you suggest I should remove the handle argument already in this PR (for our fork) or just in regards to upstreaming?

Reply:
Right ok, yeah I am not sure. I feel like given we have all the extra handle changes it's probably better to have our changes be consistent in supporting different handles?

Reply:
We never update the aggregation labels on the public metrics endpoint though and we've enforced this via the filesystem. I don't really mind either way though.

Reply:
True, can also always add it again if needed.

Have removed.